src/github/client/github-client.ts

```diff
@@ -288,4 +288,18 @@ export default class GitHubClient {
 				}
 			});
 	}
+
+	public async updateIssueComment({ owner, repo, comment_id, body }: Octokit.IssuesUpdateCommentParams): Promise<AxiosResponse<Octokit.IssuesUpdateCommentResponse>> {
```

Comment:
There is a `private async patch` method once you merge main that you can reuse here.